### ISSUE TITLE
No Video Support :(

### ISSUE THREAD

**Issue by FlynnD273** [KristianKalm/HangoutsDownloader#2]
I realize that this was made quite a while ago, so you might not even be changing this anymore. But first off, this is exactly what I wanted, and the python code is really easy to understand. Thank you. But, for videos, I guess Google changed the way that they're stored? Now the videos have a url that points to the thumbnail, and the media type tag is set to VIDEO. Then there's streaming tokens and a download link url. Do you happen to know how to get video to download too?

**Comment by BanCrash** [comment]
Yep, I think it's an issue with Google. I checked and the video url says:

>  404. That’s an error.
> 
> The requested URL ***** was not found on this server. That’s all we know.

The only link that is working is the thumbnail link...
